### DISCUSSION TITLE
Can you make it possible to download model from civitai

### DISCUSSION THREAD

**Discussion by Relsuu** [etherealxx/etherportal-webui-colab#5] [Ideas]
Just like the title says it would be e great feature to be able to download model from civitai.....maybe

**Comment by etherealxx** [comment]
> Just like the title says it would be e great feature to be able to download model from civitai.....maybe

there's an extension inside the webui called Batchlinks Downloader, you can use that to download from civit.👍✨

Currently i'm busy developing that extension. But when i'm done, i'll update this colab again.
